web/styles/drafts.css

```diff
@@ -1,103 +1,81 @@
+        justify-content: space-between;
+        gap: 5px;
 
-    .draft-info-box {
-        width: 100%;
-        margin-bottom: 10px;
+        .removed-drafts {
+            text-align: left;
+            margin-left: 25px;
 
-        &.active {
-            outline: 2px solid hsl(215deg 47% 50%);
-            border-radius: 7px;
+            @media (width < $lg_min) {
+                text-align: center;
+                margin-left: 0;
+            }
         }
 
-        .message_row {
-            border-radius: 0 0 7px 7px;
-            border: 1px solid var(--color-message-list-border);
-            border-top: 0;
-        }
+        .delete-drafts-group {
+            display: flex;
+            justify-content: flex-end;
+            align-items: center;
 
-        .messagebox-content {
-            grid-template-rows: auto;
-            grid-template-columns: auto max-content;
-            padding: 10px;
+            .delete-selected-drafts-button {
+                margin-right: 10px;
 
-            .message_content {
-                grid-column: 1 / 2;
-                /* to space from restore draft button */
-                margin-right: 5px;
+                &:focus {
+                    background-color: hsl(0deg 0% 93%);
+                }
             }
 
-            .message_top_line {
-                grid-column: 2 / 3;
+            .select-drafts-button {
+                display: flex;
+                align-items: center;
+                justify-content: center;
+                margin-right: 25px;
+                padding-right: 10px;
+                padding-left: 10px;
+                gap: 5px;
+
+                &:focus {
+                    background-color: hsl(0deg 0% 93%);
+                }
             }
-        }
-
-        .messagebox {
-            cursor: auto;
-            box-shadow: none;
-            border-radius: 0 0 7px 7px;
-        }
 
-        .draft_controls {
-            display: inline-block;
-            font-size: 0.9em;
-
-            [data-tippy-root] {
-                width: max-content;
-                word-wrap: unset;
+            .select-state-indicator {
+                width: 15px;
             }
 
-            .restore-draft {
-                cursor: pointer;
-                margin-right: 5px;
-                color: hsl(170deg 48% 54%);
-                opacity: 0.7;
-
-                &:hover {
-                    opacity: 1;
-                }
+            @media (width < $lg_min) {
+                margin-top: 5px;
+                width: 100%;
             }
+        }
 
-            .delete-draft {
-                cursor: pointer;
-                margin-left: 5px;
-                color: hsl(357deg 52% 57%);
-                opacity: 0.7;
+        @media (width < $lg_min) {
+            flex-direction: column;
+        }
+    }
 
-                &:hover {
-                    opacity: 1;
-                }
-            }
+    .drafts-list {
```

Comment:
Is this a stale selector? I'm not seeing a `.removed-drafts` inside of `.drafts-list` in the DOM or the template here.